src/Scripting/CSharp/Hosting/ObjectFormatter/CSharpPrimitiveFormatter.cs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using Microsoft.CodeAnalysis.Scripting.Hosting;
+
+namespace Microsoft.CodeAnalysis.CSharp.Scripting.Hosting
+{
+    using static ObjectFormatterHelpers;
+
+    public class CSharpPrimitiveFormatter : CommonPrimitiveFormatter
```

Comment:
Don't expose types with prefix Common.  It should just be named PrimitiveFormatter